src/tests/stdio/fgets_test.c

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file
+ * @brief
+ *
+ * @author  Nastya Nizharadze
+ * @date    08.07.2019
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("stdio/fgets test");
+
+/* File's content is
+* qwer\n
+* asdf\n
+* \n
+* zxcv
+*/
+#define TEST_FILE_NAME       "/test_fgets.txt"
+#define TEST_FILE_NAME_EMPTY "/test_fgets_empty.txt"
+
+#define STR_SIZE_TEST (6 * sizeof(char))
+
+TEST_CASE("fgets should read lines with \\n at the end") {
+	FILE *fp = fopen(TEST_FILE_NAME, "r");
```

Comment:
Please separate variable declarations and initializations

Reply:
Done.